TramsDataApi.Test/Controllers/AcademyConversionProjectsControllerV2Tests.cs

```diff
@@ -193,7 +230,15 @@ public async Task GetConversionProjects_ReturnsResponseWithListOfAcademyConversi
 				new Mock<IGetAcademyConversionProjectFilterParameters>().Object,
 				_mockLogger.Object);
 
-			var result = await controller.GetConversionProjects(null, title: null, urn: urn, deliveryOfficers: null, regions: projectRegionEstablishmentURNs);
+            GetAcademyConversionSearchModel searchModel = new GetAcademyConversionSearchModel
```

Comment:
Small thing: I'd be tempted to pass these into the constructor so that you don't need to add all these additional lines.

Reply:
Have just pushed a change to do this. I am resolving tests that have broken as a result.